Formula/podman.rb

```diff
@@ -27,6 +29,16 @@ def install
     fish_completion.install "completions/fish/podman.fish"
   end
 
+  def caveats
+    if Hardware::CPU.arm?
+      <<~EOS
+        Podman machine on arm requires unmerged upstream qemu patches.
+        Please refer to the podman machine doc for more information:
+        https://github.com/containers/podman/blob/main/docs/tutorials/mac_experimental.md
+      EOS
+    end
+  end
```

Comment:
We only carry `caveats` that are Homebrew-specific, so I'm not sure this one is suitable.

Reply:
I guess it's home-brew specific that the qemu patches are not in the homebrew packaged qemu for m1 only? let me know if this caveat is still okay, otherwise, I'll change it and see what I can do.

Reply:
That would be true about any `qemu` package not containing the appropriate patches, whether it's packaged by Homebrew or not. This is a caveat that sounds a lot like something that could (and likely should) be in upstream documentation. We don't add things like that to formulae -- we would end up with too many for our small team to maintain.